### PR TITLE
Update dints bundle

### PR DESCRIPTION
**Pull request by dongyang0122**
### Description
Update dints bundle with re-trained checkpoints with new implementation of dints in monai.

### Status
Ready

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/pancreas_ct_dints_segmentation/configs/inference.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ input_channels: 1
 output_classes: 3
 arch_ckpt_path: "$@bundle_root + '/models/search_code_18590.pt'"
 arch_ckpt: "$torch.load(@arch_ckpt_path, map_location=torch.device('cuda'))"
-bundle_root: "/workspace/MONAI/model-zoo/models/pancreas_ct_dints_segmentation"
+bundle_root: "."
 output_dir: "$@bundle_root + '/eval'"
```
Comment (yiheng-wang-nv) [inline models/pancreas_ct_dints_segmentation/configs/inference.yaml:10]: cc @dongyang0122 , I changed the bundle_root into ".", this is to align with other bundles as discussed in the ticket: https://github.com/Project-MONAI/model-zoo/issues/247

**Review by yiheng-wang-nv — APPROVED** [review]

**Comment by yiheng-wang-nv** [comment]
This PR requires a new MONAI tag version, and after released it can be merged.

File: models/pancreas_ct_dints_segmentation/configs/inference.yaml (hunk 1 above)
Comment (dongyang0122) [reply models/pancreas_ct_dints_segmentation/configs/inference.yaml:10]: sounds good.

**Comment by yiheng-wang-nv** [comment]
cc @dongyang0122 , I did some changes to use new workflow to align with other bundles.

**Review by Nic-Ma — APPROVED** [review]

**Comment by dongyang0122** [comment]
@yiheng-wang-nv The changes look good to me. Thanks!

**Pull request merged**